e2e/pages/adf/demo-shell/process-services/taskDetailsPage.ts

```diff
@@ -0,0 +1,46 @@
+/*!
+ * @license
+ * Copyright 2019 Alfresco Software, Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { by, element } from 'protractor';
+import { Util } from '../../../../util/util';
+
+export class TaskDetailsPage {
+
+    taskDetailsHeader = element(by.css('ng-component h4'));
```

Comment:
Is it possible find a better hook? like an id?

Reply:
not sure :) maybe you can suggest..
<img width="1337" alt="screenshot 2019-02-21 at 11 30 34" src="https://user-images.githubusercontent.com/45559635/53166098-50feff00-35cc-11e9-9781-0adcb232ac43.png">

Reply:
use the data-automantion-id go-back>h4

Reply:
I have added a new data automation id for the task details header.